packages/core/src/navigation/navigation-group.element.ts

```diff
@@ -0,0 +1,198 @@
+
+import { html, internalProperty, LitElement, PropertyValues } from 'lit-element';
+
+import {
+  Animatable,
+  animate,
+  AnimationNavigationGroupOpenName,
+  baseStyles,
+  createId,
+  event,
+  EventEmitter,
+  i18n,
+  I18nService,
+  id,
+  property,
+  querySlot,
+  querySlotAll,
+  reverseAnimation,
+  syncProps,
+} from '@cds/core/internal';
+import { styles } from './navigation-group.element.css.js';
+import { NavigationLayout } from './interfaces/navigation-layout.interface.js';
+import { DEFAULT_NAVIGATION_LAYOUT } from './utils/index.js';
+import { CdsNavigationItem } from './navigation-item.element';
+import { CdsNavigationStart } from './navigation-start.element';
+
+export const CdsNavigationGroupTagName = 'cds-navigation-group';
+
+/**
+ *
+ * ```typescript
+ * import '@cds/core/navigation/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-navigation-group>
+ *
+ * </cds-navigation-group>
+ * ```
+ *
+ * @element cds-navigation-group
+ * @slot
+ */
+@animate({
+  expanded: {
+    true: AnimationNavigationGroupOpenName,
+    false: reverseAnimation(AnimationNavigationGroupOpenName),
+  },
+})
+export class CdsNavigationGroup extends LitElement implements Animatable {
+  @property({ type: String })
+  cdsMotion = 'on';
+
+  @event()
+  cdsMotionChange: EventEmitter<string>;
+
+  @event()
+  expandedChange: EventEmitter<boolean>;
+
+  /**
+   * @desc
+   * Associate the cds-navigation-button in this template with projected into the
+   * aria-labelledby wrapper region.
+   *
+   * @private
+   */
+  @id() navigationGroupId: string;
+
+  @i18n() i18n = I18nService.keys.navigation;
+
+  /**
+   * @description
+   * Getter method for a reference to the selector cds-navigation-group > cds-navigation-start
+   * This lets each group flag its cds-navigation-start element and sync that info down. This is
+   * needed because cds-navigation-start elements can be used at the root level and inside
+   * cds-navigation-group elements.
+   *
+   * @private
+   */
+  @internalProperty()
+  private get isGroupStart(): boolean {
+    return !!this.groupStart;
+  }
+
+  /**
+   * @desc
+   * The element id is used to update cds-navigation aria-activedescendent when a group is the current
+   * active item.
+   */
+  @property({ type: String, reflect: true, attribute: 'id' })
+  id: string = createId();
+
+  @internalProperty()
+  protected layout: NavigationLayout = DEFAULT_NAVIGATION_LAYOUT;
+
+  @property({ type: Boolean, reflect: true })
+  expanded = false;
+
+  @property({ type: Boolean })
+  active: boolean;
+
+  /**
+   * @desc
+   * The value of this property is passed down to start and item children. It is used to query for visible items when
+   * managing focus with key events in the root cds-navigation element.
+   *
+   * @private
+   */
+  // Note: eslint-disable  @typescript-eslint/no-unused-vars isn't ignoring the line
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+  // @ts-ignore
+  private expandedGroup = false;
```

Comment:
The issue stems from the fact that I need this to sync the property down but I never actually use it anywhere. If I use it in the template the eslint error goes away because its getter is called. That rule seems to want the property to do both get and set or it complains. 

I updated the note to be more descriptive.